enzy_htp/_config/rosetta_config.py

```diff
@@ -32,6 +32,9 @@ class RosettaConfig(BaseConfig):
         PARAMS_SCRIPT: str() of the parameter generation file needed for RosettaLigand.
     """
 
+    REMODEL:str="$ROSETTA3/source/bin/remodel.linuxgccrelease"
```

Comment:
This does not work for my Rosetta install which the name is remodel.static.linuxgccrelease